src/test/java/uk/gov/ons/ctp/integration/contactcentresvc/util/PgpEncryptTest.java

```diff
@@ -0,0 +1,100 @@
+package uk.gov.ons.ctp.integration.contactcentresvc.util;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.google.common.io.CharStreams;
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.util.ArrayList;
+import java.util.Collection;
+import org.junit.Test;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.core.io.Resource;
+
+public class PgpEncryptTest {
+  private static final String PUBLIC_KEY_1 = "pgp/key1.asc";
+  private static final String PUBLIC_KEY_2 = "pgp/key2.asc";
+
+  static final String PRIVATE_KEY_1 = "pgp/priv-key1.asc";
+  static final String PRIVATE_KEY_2 = "pgp/priv-key2.asc";
+
+  public static final String PASS_PHRASE = "Good Golly Miss Molly";
+  public static final String PASS_PHRASE2 = "Bless My Soul";
+
+  static final String TEST_STRING =
+      "God grant me the serenity to accept the things I cannot change, "
+          + "Courage to change the things I can, "
+          + "and Wisdom to know the difference.";
```

Comment:
What if you can change something but God doesn't want you to?

Reply:
Don't know!  I inherited the test string from Martin!